templates/collection.yaml

```diff
@@ -217,20 +221,31 @@ Parameters:
       - false
   InvokeSnapshotOnStartEnabled:
     Type: String
-    Default: true
+    Default: 'true'
     Description: >-
       Toggle invocation of snapshot from Cloudformation. This can be useful for
       debug purposes if the lambda fails to complete successfully.
     AllowedValues:
       - true
       - false
+  MetricsEnabled:
+    Type: String
+    Default: 'true'
+    Description: >-
+      Whether to collect CloudWatch Metric data or not. Only disable if you already submit CloudWatch Metric data separately.
+    AllowedValues:
+      - true
```

Comment:
If `Default` is a string seems like these should be too

Reply:
changed default to bool